### PR TITLE
Cromwell script: Handle error if cromshell is not running

### PR DESCRIPTION
**Pull request by NehaBroad**
\<your comments for this pull request go here\>

Unfortunately we don't have automated testing configured for the code in this
repository yet so we set up this checklist as an *automatic reminder*:

- [ ] Ensure that the smoke tests pass using the current (or upcoming) CDR
- [ ] Update documentation relevant to this pull request

Questions? See [CONTRIBUTING.md](https://github.com/all-of-us/workbench-snippets/blob/master/CONTRIBUTING.md)
or file an issue so that we can get it documented!

### CODE REVIEW

File: py/py_cromwell_setup.py [hunk 1]
```diff
@@ -30,15 +30,16 @@ def check_for_app(env):
...
 
 def get_app_details(env, app_name):
     get_app_url = f'{env["leonardo_url"]}/api/google/v1/apps/{env["google_project"]}/{app_name}'
     print('start')
     r = requests.get(
         get_app_url,
         params={
-            'includeDeleted': 'true'
+            'includeDeleted': 'true',
+            'role': 'creator'
```
Comment (jmthibault79) [inline py/py_cromwell_setup.py:42]: does this additionally filter it to only my user's apps?

File: py/py_cromwell_setup.py [hunk 2]
```diff
@@ -75,13 +76,13 @@ def find_app_status(env):
     print(f'Checking status for CROMWELL app')
     app_name, app_status, proxy_url = check_for_app(env)
 
-    print(f'app_name={app_name}; app_status={app_status}')
+    configure_cromwell(env, proxy_url)
```
Comment (jmthibault79) [inline py/py_cromwell_setup.py:79]: previously we would not configure_cromwell if there was no app name.  why is it ok to do so now?

File: py/py_cromwell_setup.py [hunk 3]
```diff
@@ -63,10 +64,10 @@ def configure_cromwell(env, proxy_url):
      print('Updating cromwell config')
      file = f'{str(Path.home())}/.cromshell/cromshell_config.json'
      configuration = {
-        'cromwell_server': proxy_url.split("swagger/", 1)[0],
+        'cromwell_server': proxy_url.split("swagger/", 1)[0] if proxy_url else "",
         'requests_timeout': 5,
         'gcloud_token_email': env['user_email'],
-        'referer_header_url': env['referer']
+        'referer_header_url': env['leonardo_url']
```
Comment (jmthibault79) [inline py/py_cromwell_setup.py:70]: why this change?

**Review by jmthibault79 — CHANGES_REQUESTED** [review]
Looks like there are several different changes here.  Please explain them.

File: py/py_cromwell_setup.py (hunk 1 above)
Comment (NehaBroad) [reply py/py_cromwell_setup.py:42]: yes, ticket ref: https://broadworkbench.atlassian.net/browse/IA-3823

File: py/py_cromwell_setup.py (hunk 3 above)
Comment (NehaBroad) [reply py/py_cromwell_setup.py:70]: The refere_header should point to leonardo since thats where the request is generating from, using refere(bascially AOU URL) was resulting in authorization error

File: py/py_cromwell_setup.py (hunk 2 above)
Comment (NehaBroad) [reply py/py_cromwell_setup.py:79]: so in case user had a cromwell server running the config was pointing to say serverA. that server is now deleted so rather than cromwell configuration pointing to old serverA i believe it should reflect the true state i,.e  empty string.

**Review by jmthibault79 — APPROVED** [review]
Yes this does detect when Cromwell is not running
